ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml

```diff
@@ -48,5 +49,6 @@
             </rico:descriptiveNote>
          </rico:PerformanceRelation>
       </rico:agentIsTargetOfPerformanceRelation>
+      <rico:performsOrPerformed rdf:nodeID="_Activity-b" />
```

Comment:
Same as above: the following line can be added:
`<rico:hasOrHadOccupationOfType rdf:resource="occupationType/FRAN_RI_010-FRAN_RI_010-d3nyub73ic-1g5es55lb2lpf"/>`